package/MDAnalysis/lib/formats/include/readdcd.h

```diff
@@ -136,7 +136,7 @@ static int write_dcdheader(fio_fd fd, const char *remarks, int natoms,
  * Output: 0 on success, negative error code on failure.
  * Side effects: coordinates are written to the dcd file.
  */
-static int write_dcdstep(fio_fd fd, int curstep, int curframe, 
```

Comment:
My other assumption was that long-term we'd replace the plugin C code, and maybe even try to have something that becomes progressively faster--part of the objective to have independent format readers, etc. Maybe after asv or some benchmarking library is used to make sure our performances are improving over the project lifespan.